apps/client/src/common/utils/eventsManager.ts

```diff
@@ -24,6 +24,7 @@ export const cloneEvent = (event: OntimeEvent): ClonedEvent => {
     skip: event.skip,
     colour: event.colour,
     revision: 0,
+    delay: 0, //force delay to zero, will be populated by the server
```

Comment:
I am not sure this warrants the cognitive load of having even to think about it.
we could just 
a) remove the comment 
b) copy the delay of the cloned event